worlds/psychonauts/README.md

```diff
@@ -0,0 +1,19 @@
+# Psychonauts_AP
+
+This project is a work in progress Archipelago World for the Psychonauts Randomizer.
+
+Please read docs/setup_en.md for AP world setup and installation instructions.
+
+Make sure to use the latest version of the PsychoRando AP Branch.
+
+Randomizer Mod Here:
+https://github.com/Akashortstack/PsychoRando/releases
+
+Requires Astralathe to run:
+https://gitlab.com/scrunguscrungus/astralathe/-/releases
+
+Setup Guide and list of changes from vanilla game here:
+https://docs.google.com/document/d/1b7QOnOLmTSvdC7A1YK3bsSmhtSOsAMs0XF5j-tyE6Zw/edit
+
+PopTracker Pack created by Mysteryem here:
+https://github.com/Mysteryem/Psychonauts_AP_PopTracker/releases
```

Comment:
I think it would be useful to link that in the setup guide or the game info page. It will be easier to access for users